core/example/benchmark/CMakeLists.txt

```diff
@@ -3,14 +3,8 @@ if(Cabana_ENABLE_Serial)
   target_link_libraries(NeighborListMDPerfTest cabanacore)
 endif()
 
-# PeakFlops uses x86 asm volatile
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "([xX]86|[aA][mM][dD]64)")
-  add_executable(PeakFlops Cabana_peakflops.cpp)
-  target_link_libraries(PeakFlops cabanacore)
-endif()
-
-if(Cabana_ENABLE_MPI AND Cabana_ENABLE_Cuda AND Cabana_ENABLE_OpenMP)
```

Comment:
This logic was the correct one - looks like this got changed in the rebase

Reply:
Do you mean we want:
`if(Cabana_ENABLE_MPI AND Cabana_ENABLE_Cuda AND Cabana_ENABLE_OpenMP)`
over 
`if(Cabana_ENABLE_MPI AND Cabana_ENABLE_Cuda AND (Cabana_ENABLE_Serial OR Cabana_ENABLE_OpenMP OR Cabana_ENABLE_Pthread) )`
or something else?

Reply:
Yes we want `if(Cabana_ENABLE_MPI AND Cabana_ENABLE_Cuda AND Cabana_ENABLE_OpenMP)`

Reply:
fixed